### ISSUE TITLE
Boolean query works only with 0/1

### ISSUE THREAD

**Issue by androidfanatic** [satyan/sugar#482]
This return a list of size 0. 

    Model.find(Model.class, "favorite = ?", "true") 

where favorite is a boolean type inside Model class.

I read a bit about the issue and turns out this works fine (using 1/0 for true/false)

    Model.find(Model.class, "favorite = ?", "1") 

It'd be nice to be able to query boolean fields using true/false string instead of 0/1.

**Comment by adiyatmubarak** [comment]
This is also happened in query builder.
```
Select.from(Model.class).where(Condition.prop("is_From").eq(true)).count()
```

I have to change `true` with `"1"`

**Comment by jonisaa** [comment]
We will take care of this to add as feature of sugar.

**Comment by JonatanSalas** [comment]
This is now solved in PR #553

**Comment by mehtaronak** [comment]
This is still not working, Can anyone verify ?
